jellyfin_mpv_shim/webclient_view/__init__.py

```diff
@@ -54,10 +54,13 @@ def stop(self):
         self.join()
 
     def run(self):
+        print(os.path.dirname(sys.executable))
         app = Flask(
             __name__,
             static_url_path="",
-            static_folder=get_resource("webclient_view", "webclient"),
```

Comment:
Update the get_resource function to properly resolve the path on macOS as it should work for any packaged files. You may need an extra check if you can't include it with the Python module.

Reply:
I've just updated the get_resource function with an extra check for Darwin.